src/main/scala/is/hail/rvd/OrderedRVType.scala

```diff
@@ -38,6 +39,18 @@ class OrderedRVType(
   val pkInKOrd: UnsafeOrdering = OrderedRVType.selectUnsafeOrdering(kType, pkKFieldIdx, kType, pkKFieldIdx)
   val kRowOrd: UnsafeOrdering = OrderedRVType.selectUnsafeOrdering(kType, (0 until kType.size).toArray, rowType, kRowFieldIdx)
 
+  // FIXME: replace when physical types arrive
+  // parameterized type is a hack to permit exclusion of entries
+  def makeValueStruct(t: TStruct = rowType): (RegionValue => Row) = {
```

Comment:
You don't ever set the parameter.  Can simplify.

Reply:
just deleted this in favor of an allocation-free solution. 👍